web-client/src/pages/role.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Rename the file from role.tsx to RolePage.tsx to be consistent with the rest of the project.

Reply:
Renamed to RoleInfoPage.tsx